### PR TITLE
Bug fix for removing keys from `validation.attributes` in translation managers

### PR DESCRIPTION
**Pull request by andrey-helldar**
### Problem

Translation managers don't carry keys from an attribute in an app on update.

### Reason

The transfer is done by reading the keys. There are no `attributes` keys in the `source/validation.php` and `source/validation-inline.php` files.

### Solution

Add keys to the source files.

### CODE REVIEW

File: source/validation.php [hunk 1]
```diff
@@ -154,5 +154,39 @@
...
+        'hour'                  => 'Hour',
+        'last_name'             => 'Last Name',
+        'message'               => 'Message',
+        'minute'                => 'Minute',
+        'mobile'                => 'Mobile',
+        'month'                 => 'Month',
+        'name'                  => 'Name',
+        'password'              => 'Password',
+        'password_confirmation' => 'Password Confirmation',
+        'phone'                 => 'Phone',
```
Comment (luisprmat) [inline source/validation.php:180]: Could you add the key `photo` that is used in [Laravel Jetstream](https://github.com/laravel/jetstream/blob/2.x/stubs/app/Actions/Fortify/UpdateUserProfileInformation.php#L24)?

**Review by luisprmat — CHANGES_REQUESTED** [review]

File: source/validation.php (hunk 1 above)
Comment (andrey-helldar) [reply source/validation.php:180]: Agree. Added.

Comment (andrey-helldar) [reply source/validation.php:180]: See also https://github.com/Laravel-Lang/lang/pull/1862

**Comment by caouecs** [comment]
I can merge ?

**Comment by andrey-helldar** [comment]
@caouecs, yes

**Comment by caouecs** [comment]
done

**Comment by andrey-helldar** [comment]
Thank you 😊

**Pull request merged**